directory_header_footer/context_processors.py

```diff
@@ -14,6 +16,77 @@ def sso_processor(request):
     }
 
 
+def urls_processor(request):
+    return {
+        'great_home': getattr(
```

Comment:
can you make this dict nested - so it returns this:
```
return {
    'header_footer_urls': {
        'great_home': ...
    }
}
```

that way when the templates use the variables provided by `urls_processor`, we know what is providing the values - because the template will do
```
{{ header_footer_urls.great_home  }} 
```

this will help create a "paper trail" for devs to follow to understand the source of the data.

Reply:
oh cool. will fix this